sn06.yml

```
@@ -39,6 +39,7 @@
       - "*.interactivetoolentrypoint.interactivetool.usegalaxy.eu"
       - "*.interactivetoolentrypoint.interactivetool.live.usegalaxy.eu"
       - "*.interactivetoolentrypoint.interactivetool.test.usegalaxy.eu"
+      - "*.aquainfra.usegalaxy.eu"
```

```suggestion
      - "*.aquainfra.usegalaxy.eu"
      - "*.ecology.usegalaxy.eu"
```

That will also make @yvanlebras happy.

---

Oh, they named themself `aqua`.

---

Is it what I think ? ;)